### ISSUE TITLE
Automate posting of release notes to Slack

### ISSUE THREAD

**Issue by coreycarvalho** [department-of-veterans-affairs/notification-api#1972]
## User Story - Business Need

Currently, at the end of every release, a developer manually copies and pastes the release notes into Slack. In order to fully automate the CI/CD pipeline, we need to remove the manual step of copy/pasting the release notes into Slack.

- [ ] Ticket is understood, and QA has been contacted (if the ticket has a QA label).

### User Story(ies)

**As a**    VA Notify developer
**I want**  automatic release note posting to Slack
**So that** I don't have to copy and paste them manually.

## Acceptance Criteria

- [x] When the CI/CD pipeline finishes, and a new version is released to production, release notes are automatically sent to Slack
  - [x] #va-notifications-va-gov-program (Oddball)
  - [ ] #va-notify-public (Office of CTO @ VA)

## QA Considerations
 - work with Dev on any formatting/language needs for the posts that get made to Slack.

## Potential Dependencies

If we want to post to the Oddball workspace, we will need an API token or a Slackbot set up to be able to do that. Will have to reach out to somebody on the Oddball side for that.

**Comment by npmartin-oddball** [comment]
@cris-oddball Can you track down the anomaly detection ticket and set as a blocker to this ticket? Thank you!

**Comment by cris-oddball** [comment]
The anomaly ticket is now listed as a blocker.

@coreycarvalho we also need to understand when the release notes would be posted to slack? via another pause for approval step in the pipeline or automatically based on the results of anomaly detection? In other words, we only want to post release note when we are satisfied that the release was a success.

**Comment by coreycarvalho** [comment]
Reached out to @k-macmillan to get a Slackbot webhook URL set up via https://api.slack.com/messaging/webhooks on the Oddball side. Going to reach out to VFS platform support to see if I can get a webhook set up on their end.

Until then, there isn't anything I can do with this work. Putting this down and grabbing a new ticket until I get an answer.

**Comment by coreycarvalho** [comment]
The app and webhook have both been created on the Oddball side. Functionality has been tested successfully! Just waiting on the VA slack admins to set this up on their end.

**Comment by coreycarvalho** [comment]
Automated posting to Oddball's Slack is done. Waiting on the VA to get us a Slack webhook URL to use.

**Comment by coreycarvalho** [comment]
Closing this ticket in favor of opening up a new one to address the VA Slack side of the house. We need to wait for them to give me a webhook to the desired Slack channel, and we don't know how long that will take. Doesn't make sense to leave this ticket open until that happens.
